### PR TITLE
Feat/ Venue: publication chair

### PR DESCRIPTION
**Pull request by celestemartinez**
Addresses #1589

Needs https://github.com/openreview/openreview-api-v1/pull/2906 and https://github.com/openreview/openreview-web/pull/1530

### CODE REVIEW

File: openreview/venue_request/venue_request.py [hunk 1]
```diff
@@ -1449,14 +1455,16 @@ def setup_request_form(self):
                             'values-copied': [
                                 self.support_group.id,
                                 '{signatures}',
-                                '{content["program_chair_emails"]}'
+                                '{content["program_chair_emails"]}',
+                                '{content["publication_chair_email"]}'
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:1459]: can you debug the UI and see what happens when we post the note?

is it being sent a "null" value? if that is the case, we should check in view.js where we are getting the copied values and if the value is null then don't send it.

File: openreview/venue/webfield/publicationChairWebfield.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// Webfield component
```
Comment (celestemartinez) [inline openreview/venue/webfield/publicationChairWebfield.js:1]: @haroldrubio Here I am adding the webfield, it only has an 'Accepted Submissions' tab, so if publication chairs are readers of accepted submissions they will see them here.

**Review by melisabok — APPROVED** [review]

**Comment by melisabok** [comment]
Can I merge this?

**Comment by celestemartinez** [comment]
> Can I merge this?

Yes, this is ready!

File: openreview/venue/group.py [hunk 3]
```diff
@@ -433,6 +433,29 @@ def create_ethics_chairs_group(self):
                 ethics_chairs_group.web = content
                 self.post_group(ethics_chairs_group)
 
+    def create_publication_chair_group(self):
+        venue_id = self.venue_id
+
+        publication_chair_group_id = self.venue.get_committee_id('Publication_Chair')
```
Comment (melisabok) [inline openreview/venue/group.py:439]: I think the group name should be in plural, there can be more than one publication chair.

**Pull request merged**